### ISSUE TITLE
Add support for `Cc`, `Bcc` and `Reply-To` fields

### ISSUE THREAD

**Issue by CedX** [benmerckx/smtpmailer#19]
I don't have the impression that we can provide anything other than the `from` and `to` fields to specify the addresses of a message (cf. `Email` typedef). I suppose that we need to use the `headers` field for this purpose but it would be much easier if this was directly supported by the library.

**Comment by benmerckx** [comment]
Yeah, these need to be added. Haven't had the need for them so far so have been putting it off

**Comment by CedX** [comment]
No problem, it's not blocking. And thank you for your efforts 👍
